README.md

```diff
@@ -1,2 +1,89 @@
 # Portfolio
 Set up a portfolio.
+
+Built With
+
+Tech Stack
+
+Client
+    Node.js
+
+Server
+    
+Database
+
+Key Features
+
+    Display a portfolio of Projects worked upon and Achievements.
+
+Live Demo
+https://github.com/lenkon/Portfolio
+
+Getting Started
+
+To get a local copy up and running, follow these steps.
+
+Prerequisites
+
+In order to run this project you need:
+
+Setup
+
+Clone this repository to your desired folder:
+
+Install
+
+Install this project with:
+
+Usage
+
+To run the project, execute the following command:
+
+Run tests
+
+To run tests, run the following command:
+
+Deployment
+
+You can deploy this project using:
+
+Authors
+
+Author1
+
+    GitHub: @lenkon
+    Twitter: 
+    LinkedIn: 
+
+Author2
+
+    GitHub: 
+    Twitter:
+    LinkedIn:
+
+
+Future Features
+
+    1. Add more projects.
+    2. Add milestones and achievements.
+    3.
+
+Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+Show your support
+
+If you like this project I would like to encourage One and All to kindly support the project by sharing it.
+
+Acknowledgments
+
+I would like to thank...
+
+FAQ
+
+License
+
+This project is MIT licensed.
```

Comment:
- [ ] You should add a descriptive README file to your project. please make every content heading so it would be better to understand and it shows your professionalism. For more details see this [Link](https://github.com/microverseinc/readme-template)

Reply:
@roshan-bajgain Made the changes to make the heading contents look more readable. Thanks again.